src/components/package-port/CodeEditor.tsx

```diff
@@ -251,16 +253,18 @@ export const CodeEditor = ({
 
           // Generate TypeScript declarations for TypeScript/TSX files
           if (currentFile.endsWith(".ts") || currentFile.endsWith(".tsx")) {
-            const { outputFiles } = env.languageService.getEmitOutput(
-              currentFile,
-              true,
-            )
-            const dtsFile = outputFiles.find((file) =>
-              file.name.endsWith(".d.ts"),
-            )
-            if (dtsFile?.text && onDtsChange) {
-              onDtsChange(dtsFile.text)
-            }
+            try {
+              const { outputFiles } = env.languageService.getEmitOutput(
+                currentFile,
+                true,
+              )
+              const dtsFile = outputFiles.find((file) =>
+                file.name.endsWith(".d.ts"),
+              )
+              if (dtsFile?.text && onDtsChange) {
+                onDtsChange(dtsFile.text)
+              }
+            } catch {}
```

Comment:
why is this here? silencing errors is _really really bad_ without an explanation!!!